src/lib/tree/Constants.sol

```diff
@@ -15,7 +15,13 @@ library Constants {
     bytes1 internal constant LEAF_PREFIX = 0x00;
     bytes1 internal constant NODE_PREFIX = 0x01;
 
-    /// @dev Parity share namespace ID
-    NamespaceID internal constant PARITY_SHARE_NAMESPACE_ID =
-        NamespaceID.wrap(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF);
+    /// @dev Parity share namespace in bytes.
+    bytes29 internal constant PARITY_SHARE_NAMESPACE_BYTES =
```

Comment:
Is this constant ever used?

Reply:
it's just left there for reference on what the parity share namespace is. I can remove it if you prefer

Reply:
yeah, please remove since it's not used anywhere.

Reply:
[354aadf](https://github.com/celestiaorg/quantum-gravity-bridge/pull/175/commits/354aadf24a74f855f6a3d4b9062e4c8cfa81884c)